Configurateur/front/lib/screens/not-found/not_found.dart

```diff
@@ -0,0 +1,142 @@
+// ignore_for_file: use_build_context_synchronously, use_full_hex_values_for_flutter_colors
+
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:footer/footer_view.dart';
+import 'package:footer/footer.dart';
+import 'package:front/components/custom_footer.dart';
+import 'package:front/components/custom_header.dart';
+import 'package:front/services/size_service.dart';
+import 'package:front/services/storage_service.dart';
+import 'package:front/services/theme_service.dart';
+import 'package:front/styles/globalStyle.dart';
+import 'package:front/styles/themes.dart';
+import 'package:go_router/go_router.dart';
+import 'package:provider/provider.dart';
+
+/// NotFoundPage
+///
+/// Home page for the web application
+class NotFoundPage extends StatefulWidget {
+  const NotFoundPage({super.key});
+
+  @override
+  State<NotFoundPage> createState() => NotFoundPageState();
+}
+
+/// NotFoundPageState
+///
+class NotFoundPageState extends State<NotFoundPage> {
+  Function() disconnectFunction = () {};
+  String? token = '';
+  String? userMail = '';
+
+  /// [Function] : Check the token in the storage service
+  void checkToken() async {
+    token = await storageService.readStorage('token');
+    storageService.getUserMail().then((value) => userMail = value);
+    setState(() {});
+  }
+
+  @override
+  void initState() {
+    super.initState();
+    checkToken();
+  }
+
+  /// [Function] : Build the NotFound page
+  @override
+  Widget build(BuildContext context) {
+    ScreenFormat screenFormat = SizeService().getScreenFormat(context);
+
+    return Scaffold(
+      body: FooterView(
+        footer: Footer(
+          child: CustomFooter(context: context),
+        ),
+        children: [
+          LandingAppBar(context: context),
+          const SizedBox(height: 200),
```

Comment:
not really responsive, why not center it and remove this huge sizedbox

Reply:
didn't worked out the way I wanted

Reply:
Center doesn't work, Container with alignement neither, Align neither,

FooterView is not made to work with this type of alignement, so we need to keep the sizedBox 200 otherwise if you have an idea, feel free to test it.